### DISCUSSION TITLE
Can you add Stacked Embeddings ?

### DISCUSSION THREAD

**Discussion by azizcu** [ThilinaRajapakse/simpletransformers#952] [Ideas]
`embedding_types = [
              TransformerDocumentEmbeddings('bert-base-uncased',fine_tune = True),
              TransformerDocumentEmbeddings('roberta-base',fine_tune = True),
              TransformerDocumentEmbeddings('microsoft/DialoGPT-small',fine_tune = True),
              ]

embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)
Stacking` many transformer model at a time . which will also work for sentence pair classification.

Like this : [Stack Embedding code line : 50 ](https://github.com/flairNLP/flair/blob/5473bfa99070de3e0c28f35ea7b89e86f2ab3b0f/flair/embeddings/token.py#L50)

**Comment by ThilinaRajapakse** [comment]
This would be nice to have but I'm not sure when I'll have time to look into it.

**Comment by Zhylkaaa** [comment]
Is there any work on this? I would love to see this feature, and also help on this
